ui/src/app/workflows/components/workflow-details/workflow-details.tsx

```diff
@@ -474,7 +549,13 @@ export const WorkflowDetails = ({history, location, match}: RouteComponentProps<
             {workflow && (
                 <SlidingPanel isShown={!!sidePanel} onClose={() => setSidePanel(null)}>
                     {parsedSidePanel.type === 'logs' && (
-                        <WorkflowLogsViewer workflow={workflow} initialPodName={podName} nodeId={parsedSidePanel.nodeId} container={parsedSidePanel.container} archived={false} />
+                        <WorkflowLogsViewer
+                            workflow={workflow}
+                            initialPodName={podName}
+                            nodeId={parsedSidePanel.nodeId}
+                            container={parsedSidePanel.container}
+                            archived={isArchivedWorkflow(workflow)}
```

Comment:
this appears to cause a subtle regression for logs for Workflows that are _both_ live and archived (e.g. ones that have not TTL'd yet): https://github.com/argoproj/argo-workflows/issues/12948#issuecomment-2061378878